### ISSUE TITLE
Is is possible to disable running commands?

### ISSUE THREAD

**Issue by shekh** [filebrowser/filebrowser#3638]
I'm scared by the possibility that someone could just run format c: from filebrowser web GUI, is this sort of big problem? Does it have a solution?
I realize that options to change shell, command runners, and list of enabled commands are available to admin. So one possible solution is to not create users with admin rights. But I'd like someone to be able to create users.
Possible solution: divide "Administrator" permission into several specific permissions, e.g. "User Management", "Global Settings".

**Comment by gtsteffaniak** [comment]
I don't believe its possible on this version for admins, but I maintain a fork which has this feature removed for these security reasons. Perhaps it would fit what you are looking for: https://github.com/gtsteffaniak/filebrowser

**Comment by shekh** [comment]
> I don't believe its possible on this version for admins, but I maintain a fork which has this feature removed for these security reasons. Perhaps it would fit what you are looking for: https://github.com/gtsteffaniak/filebrowser

Wow, great!

**Comment by xX-MrN0b0dy-Xx** [comment]
Really interesting feature request, also given [issue #3646](https://github.com/filebrowser/filebrowser/issues/3646) where some users experienced a crypto miner "injected" in some way into filebrowser...

**Comment by d-Rickyy-b** [comment]
Seems like you can disable it only via the commandline tool via the (on the website undocumented) disable-exec flag.

`filebrowser config set --disable-exec=true`

https://github.com/filebrowser/filebrowser/blob/e92dbb4bb8b7894264fbf0a48a641712c3b68766/cmd/root.go#L71

**Comment by shekh** [comment]
> Seems like you can disable it only via the commandline tool via the (on the website undocumented) disable-exec flag.

Thanks, I missed it.
However it does not work.
filebrowser config cat:
```  Exec Enabled:  false```
* Command runner setup still present in settings
* Commands are actually executed
* Console is available too (not sure if it was meant to be disabled by the option anyway)

**Comment by xiaobaidadada** [comment]
https://github.com/xiaobaidadada/filecat try it

**Comment by xX-MrN0b0dy-Xx** [comment]
> Seems like you can disable it only via the commandline tool via the (on the website undocumented) disable-exec flag.
> 
> `filebrowser config set --disable-exec=true`
> 
> [filebrowser/cmd/root.go](https://github.com/filebrowser/filebrowser/blob/e92dbb4bb8b7894264fbf0a48a641712c3b68766/cmd/root.go#L71)
> 
> Line 71 in [e92dbb4](/filebrowser/filebrowser/commit/e92dbb4bb8b7894264fbf0a48a641712c3b68766)
> 
>  flags.Bool("disable-exec", false, "disables Command Runner feature")

It would be nice to have it as an environment variable in the compose

**Comment by xX-MrN0b0dy-Xx** [comment]
I'm trying again Filebrowser, and to be sure this command execution feature get disabled immediately, I modified the entrypoint like this:
```yaml
entrypoint: ["sh","-c","/filebrowser config set --disable-exec=true && /filebrowser"]
```

**Comment by gtsteffaniak** [comment]
Looks right
